### PR TITLE
fix: pylint issues in 'tests' folder. Tests for gravity

### PR DESCRIPTION
**Pull request by i-savynikh**

### CODE REVIEW

File: _universum/lib/utils.py [hunk 1]
```diff
@@ -135,7 +135,7 @@ def import_module(name, path=None, target_name=None):
 
 def trim_and_convert_to_unicode(line):
     if isinstance(line, str):
```
Comment (miltolstoy) [inline _universum/lib/utils.py]: Remove this condition.

File: tests/conftest.py [hunk 2]
```diff
@@ -124,4 +124,3 @@ def check_output(xxx_todo_changeme):
 def detect_fails(capsys):
     yield capsys
     check_output(capsys.readouterr())
-    check_output(capsys._outerr)
```
Comment (miltolstoy) [inline tests/conftest.py]: Sure?

File: _universum/lib/utils.py (hunk 1 above)
Comment (miltolstoy) [inline _universum/lib/utils.py]: Remove this condition.

File: pytest.ini [hunk 3]
```diff
@@ -0,0 +1,3 @@
+[pytest]
```
Comment (miltolstoy) [inline pytest.ini]: Doesn't looks like pylint issues fix.

File: tests/conftest.py (hunk 2 above)
Comment (i-savynikh) [reply tests/conftest.py]: As said python, '_outerr' isn't a member of 'capsys'.

File: _universum/lib/utils.py (hunk 1 above)
Comment (i-savynikh) [reply _universum/lib/utils.py]: it will broke logic. I don't want to make refactoring until fix all tests.

File: pytest.ini (hunk 3 above)
Comment (i-savynikh) [reply pytest.ini]: This commit also contains fixes for tests

File: _universum/lib/utils.py (hunk 1 above)
Comment (miltolstoy) [reply _universum/lib/utils.py]:
```
if condition:
    pass
else if another_condition:
    do_something
```
is equal to
```
if another_condition:
    do_something
```

File: tests/conftest.py [hunk 4]
```diff
@@ -112,16 +112,14 @@ def http_check(request):
...
-    (out, err) = xxx_todo_changeme
+@pytest.fixture(autouse=True)
+def detect_fails(capsys, request):
+    yield capsys
+
+    out, err = capsys.readouterr()
     assert not err, "Stderr detected!"
     for text in ["Traceback", "Exception"]:
-        assert text not in out, text + " detected in stdout!"
+        assert text not in out, "'{}' detected in stdout!".format(text)
```
Comment (i-keliukh) [inline tests/conftest.py]: Please change to f-string

File: tests/conftest.py [hunk 5]
```diff
@@ -112,16 +112,14 @@ def http_check(request):
...
 
-
-# pylint: disable = protected-access
-@pytest.fixture(autouse=True)
-def detect_fails(capsys):
-    yield capsys
-    check_output(capsys.readouterr())
-    check_output(capsys._outerr)
+    if request.node.name == 'test_teardown_fixture_output_verification':
+        assert out == "TearDown fixture output must be handled by 'detect_fails' fixture\n"
```
Comment (i-keliukh) [inline tests/conftest.py]: Can you confirm this test failed on pytest==3.3?

File: tests/test_argument_check.py [hunk 6]
```diff
@@ -31,7 +31,7 @@ def create_settings(test_type, vcs_type):
         settings.TeamcityServer.user_id = "TeamCityUser"
         settings.TeamcityServer.passwd = "TeamCityPassword"
 
-    if vcs_type == "git" or vcs_type == "gerrit" or vcs_type == "github":
+    if vcs_type in ("git", "gerrit", "github",):
```
Comment (i-keliukh) [inline tests/test_argument_check.py]: In my opinion, lists look cleaner than tuples

File: _universum/modules/structure_handler.py [hunk 7]
```diff
@@ -191,9 +191,9 @@ def execute_steps_recursively(self, parent, variations, step_executor, skipped=F
                 else:
                     self.configs_current_number += 1
                     numbering = " [ {:>{}}/{} ] ".format(
-                        str(self.configs_current_number),
+                        self.configs_current_number,
```
[the remark this answers is not in the document]
Comment (i-savynikh) [reply _universum/modules/structure_handler.py]: from documentation:
*Returns a copy of the string where each replacement field is replaced with the string value of the corresponding argument.*

File: tests/conftest.py (hunk 5 above)
Comment (i-savynikh) [reply tests/conftest.py]: I didn't reproduce it with pylint 3.3

File: tests/conftest.py (hunk 4 above)
Comment (i-savynikh) [reply tests/conftest.py]: fixed

File: tests/test_argument_check.py (hunk 6 above)
Comment (i-savynikh) [reply tests/test_argument_check.py]: fixed

**Pull request merged**